app/pages/site.py

```diff
@@ -459,39 +461,36 @@ def calc_polygon_centroid(coordinates):
 
 
 @callback(
-    Output(component_id='interferogram',
-           component_property='layers',
+    Output(component_id='tiles',
+           component_property='url',
            allow_duplicate=True),
     Input(component_id='coherence-matrix', component_property='clickData'),
+    Input('site-dropdown', 'value'),
     prevent_initial_call=True)
-def update_interferogram(click_data):
+def update_interferogram(click_data, target_id):
     """Update interferogram display."""
+    if not target_id:
+        raise PreventUpdate
+    SITE, BEAM = target_id.split('_')
     if not click_data:
-        return 'cite:20210717_HH_20210903_HH.adf.wrp.geo'
+        return (
+            f'{TILES_BUCKET}/{SITE_INI}/{BEAM_INI}/20220821_20220914/'
+            '{z}/{x}/{y}.png'
+        )
+
     second = pd.to_datetime(click_data['points'][0]['x'])
     delta = pd.Timedelta(click_data['points'][0]['y'], 'days')
     first = second - delta
-
     first_str = first.strftime('%Y%m%d')
     second_str = second.strftime('%Y%m%d')
-    layer = f'cite:{first_str}_HH_{second_str}_HH.adf.wrp.geo'
+    layer = (
+        f'{TILES_BUCKET}/{SITE}/{BEAM}/{first_str}_{second_str}/'
+        '{z}/{x}/{y}.png'
+    )
     print(f'Updating interferogram: {layer}')
```

Comment:
An error, print statement ok temporarily, should be displayed if the rastertile does not exist

Reply:
This has been fixed